order/batch.go

```diff
@@ -51,6 +51,10 @@ const (
 	// ExtendAccountBatchVersion is the first version where accounts expiry
 	// are extended after participating in a batch.
 	ExtendAccountBatchVersion BatchVersion = 1
+
+	// UnannouncedChannelsBatchVersion is the first version where orders
+	// can set the flags to only match with announced/unannonced channels.
```

Comment:
Should we add a comment here that the batch version is used to signal readiness for setting the `private` flag when opening the channel on the asker side as well as being able to check the `private` flag in the channel acceptor on the bidder side? Otherwise we might ask ourselves "why wasn't this an order version" later down the line?